clang/include/clang/Sema/Sema.h

```diff
@@ -10082,7 +10082,9 @@ class Sema final : public SemaBase {
 
   bool SubstTemplateArgument(const TemplateArgumentLoc &Input,
                              const MultiLevelTemplateArgumentList &TemplateArgs,
-                             TemplateArgumentLoc &Output);
+                             TemplateArgumentLoc &Output,
+                             SourceLocation Loc = {},
+                             const DeclarationName &Entity = {});
```

Comment:
Changes to both declaration and implementation of `SubstTemplateArgument` doesn't seem related.

Reply:
There are a few places we used SubsType, because we only had a type, now we use SubstTemplateArgument.

SubstTemplateArgument was missing arguments for setting Instantiation location and entity names.
Adding those is needed so we don't regress in diagnostics.

Reply:
Can you add this to the description?